src/main/java/core/basesyntax/Output.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Output {
```

Comment:
Don't use verbs for class/interface names. The name 'Output' suggests it's a verb/action, while an interface should represent an ability or characteristic. Consider renaming it to something like 'Informative' or 'Describable' to better convey the intent that classes implementing this interface can provide descriptive information.